### PR TITLE
Bump versions update

### PR DESCRIPTION
**Pull request by osuleymanova**
Kotlin, gradle, AGP versions updated
target and compile SDK versions updated
BindingAdapters file modified with new imports

### CODE REVIEW

File: app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt [hunk 1]
```diff
@@ -24,6 +24,7 @@ import androidx.recyclerview.widget.RecyclerView
 import coil.load
 import com.example.android.marsphotos.network.MarsPhoto
 import com.example.android.marsphotos.overview.MarsApiStatus
+import com.example.android.marsphotos.overview.MarsApiStatus.*
```
Comment (jhansitavva) [inline app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt]: Please revert this change

File: app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt [hunk 2]
```diff
@@ -58,16 +59,17 @@ fun bindImage(imgView: ImageView, imgUrl: String?) {
...
         }
-        MarsApiStatus.ERROR -> {
+        ERROR -> {
             statusImageView.visibility = View.VISIBLE
             statusImageView.setImageResource(R.drawable.ic_connection_error)
         }
-        MarsApiStatus.DONE -> {
+        DONE -> {
             statusImageView.visibility = View.GONE
         }
```
Comment (jhansitavva) [inline app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt]: Please revert this change, why do we need this anyway? This will break the code

**Review by jhansitavva — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt [hunk 3]
```diff
@@ -58,16 +59,17 @@ fun bindImage(imgView: ImageView, imgUrl: String?) {
 @BindingAdapter("marsApiStatus")
 fun bindStatus(statusImageView: ImageView, status: MarsApiStatus?) {
```
Comment (jhansitavva) [inline app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt]: Make status non nullable
`fun bindStatus(statusImageView: ImageView, status: MarsApiStatus)`

This will fix the error you are getting.

Comment (android-dev-lxl) [reply app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt:60]: this

File: app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt (hunk 1 above)
Comment (osuleymanova) [reply app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt]: Reverted. Thanks for the comment.

File: app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt (hunk 2 above)
Comment (osuleymanova) [reply app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt]: Reverted.

File: app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt (hunk 3 above)
Comment (osuleymanova) [reply app/src/main/java/com/example/android/marsphotos/BindingAdapters.kt]: Done!

**Review by jhansitavva — APPROVED** [review]
